eidpauth/src/main/java/uk/ac/ucl/eidp/auth/jaas/EidpLoginModule.java

```diff
@@ -70,11 +70,11 @@ private void setUserService() {
 
   @Override
   public void initialize(Subject subject, CallbackHandler callbackHandler, Map<String, ?> sharedState, Map<String, ?> options) {
-    this.subject = subject;
+//    this.subject = subject;
```

Comment:
\[Checkstyle\] WARNING: Comment has incorrect indentation level 0, expected is 4, indentation should be the same level as line 74\.